packages/next/README.md

```diff
@@ -2142,6 +2142,18 @@ Note: `NODE_ENV` is properly configured by the `next` subcommands, if absent, to
 
 Note: we recommend putting `.next`, or your [custom dist folder](https://github.com/zeit/next.js#custom-configuration), in `.gitignore` or `.npmignore`. Otherwise, use `files` or `now.files` to opt-into a whitelist of files you want to deploy, excluding `.next` or your custom dist folder.
 
+### Compression
+Next.js provides [gzip](https://tools.ietf.org/html/rfc6713#section-3) compression to speed up sending content.
+
+To disable **compression** in Next.js, set the `compression` to `false` in `next.config.js`:
```

Comment:
```suggestion
In general you will want to enable compression on a HTTP proxy like nginx. 

To disable compression in Next.js, set the `compression` to `false` in `next.config.js`:
```

Reply:
> In general you will want to enable compression on a HTTP proxy like nginx.

👍 I'd also add a small statement explaining why (in high-load apps, we want to offload compression from a Node.js process to improve its performance).